elastalert/util.py

```diff
@@ -536,3 +541,19 @@ def format_string(format_config, target_value):
     else:
         return format_config % (target_value)
 
+
+def parse_hosts(host, port=9200):
+    """
+    Convert host str like "host1:port1, host2:port2" to list
+    :param host str: hostnames (separated with comma ) or single host name
+    :param port: default to 9200
+    :return: list of hosts
+    """
+    if "," in host:
+        host_list = host.split(",")
+        host_list = [("{host}:{port}".format(host=x.strip(), port=port)
+                      if ":" not in x else x.strip()) for x in host_list]
+        return host_list
+    else:
+        return ["{host}:{port}".format(host=host, port=port)]
```

Comment:
This logic is inconsistent with case above.  Does not handle `foo.org:443`.  Result will be `[ 'foo.org:443:9200']`.

There should be tests on this method.

Reply:
Good find. I'll push a change up for this bug as well, and add the additional unit test cases suggested below.